dev-db/pgpool2/pgpool2-9999.ebuild

```
@@ -25,18 +23,16 @@ RDEPEND="
 	virtual/libcrypt:=
 	memcached? ( dev-libs/libmemcached )
 	pam? ( sys-auth/pambase )
-	ssl? (
-		dev-libs/openssl:0=
-	)
+	ssl? ( dev-libs/openssl:0= )
```

```suggestion
	ssl? ( dev-libs/openssl:= )
```
the slots are pretty pointless these days

---

Done